### ISSUE TITLE
Add ResolveIPAddr and ResolveUnixAddr

### ISSUE THREAD

**Issue by paralin** [tinygo-org/net#23]
https://github.com/tinygo-org/net/blob/a79417481d37e21f29d257c28fecc503df9703e0/udpsock.go#L84

I see that ResolveTCPAddr is in, could ResolveIPAddr and ResolveUnixAddr be added?

**Comment by scottfeldman** [comment]
ResolveIPAddr looks doable.  You want to try a crack at implementing?  It should be vary similar to ResolveUDPAddr.

We don't have unix socket support in tinygo "net" package, so I would say No.

**Comment by scottfeldman** [comment]
Are you wanting to use these funcs for real networking, or just to compile?  If just for compile, we could add stubs that return "not implemented".  That would make the compiler happy.

**Comment by paralin** [comment]
I think just the stubs are fine for now!

**Comment by scottfeldman** [comment]
Oh, that's easy then.  Would you submit a PR to add those stubs?

**Comment by paralin** [comment]
Sure, on it

**Comment by paralin** [comment]
@scottfeldman here is ResolveIPAddr which actually works for some IP addresses, just not those that require accessing /etc/protocols: https://github.com/tinygo-org/net/pull/24

**Comment by paralin** [comment]
@scottfeldman Here is ResolveUnixAddr: https://github.com/tinygo-org/net/pull/25

**Comment by scottfeldman** [comment]
@paralin I reworked #24 and made a new PR #36.  Once PR #36 is merged, this issue can be closed.
